src/LoginWithAmazon.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// Options variable to request for implicit grant. Logic for assigning 'deviceSerialNumber' needs to be revisited.
+const options = {
+    scope : ['alexa:all', 'profile'],
```

Comment:
scopes

Reply:
'scope' is a request parameter that is of type list.